.goreleaser.yml

```diff
@@ -27,14 +27,43 @@ builds:
       goarch: arm64
     - goos: windows
       goarch: arm
-  binary: '{{ .ProjectName }}_v{{ .Version }}'
+  binary: 'terraform-provider-iterative'
+-
+  id: "leo"
+  main: ./cmd/leo
```

Comment:
As well, moving files from `./cmd` to `./cmd/leo` seems a bit redundant to me. 🤔 How many other binaries are we planning to generate from this repository? 😅

Reply:
By convention `cmd` is usually a container package for other commands. I don't think it hurts being explicit about command groupings.